job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/WeblogicDeployerPolicyContext.groovy

```diff
@@ -0,0 +1,123 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.Context
+
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.DEPLOYMENT_TIMER
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.LEGACY_CODE
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.REMOTE_HOST
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.SCM_CHANGE
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.UPSTREAM
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.USER
+import static javaposse.jobdsl.dsl.helpers.publisher.WeblogicDeployerPolicyContext.WeblogicDeploymentPolicies.USER_ID
+
+/**
+ * DSL Support for the weblogic-deployment-plugin's deployment policies subsection.
+ */
+class WeblogicDeployerPolicyContext implements Context {
+
+    /**
+     * Enumeration of available deployment policies.
+     */
+    static enum WeblogicDeploymentPolicies {
```

Comment:
But it's just test code. And since the enum is public, user might see and use it although it's an internal enum. So just remove it.